### DISCUSSION TITLE
Sorry to beg, but please make one to be able to install extensions.

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#94] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Preferably one that could, say, bypass chrome's restriction on foreign extensions

**Comment by synapt0** [comment, marked as answer]
probably not possible

**Reply by The-Man-Is-Me** [reply]
you would need to engener a whole other part to make chrome not update if you added a extension.

**Reply by ghost** [reply]
@The-Man-Is-Me no, just the part that disables foreign extensions

**Reply by chickennu99et** [reply]
i know a way but you need dev mode, ingots doesn't work

**Comment by phillipbruh** [comment]
check the one i found its in fixes
